pkg/innerring/innerring.go

```diff
@@ -151,6 +151,26 @@ var (
 	errDepositFail    = errors.New("notary tx has faulted")
 )
 
+type irFetcherWithNotary struct {
+	cli *client.Client
+}
+
+// InnerRingKeys fetches list of innerring keys from NeoFSAlphabet
+// role in side chain.
+func (fN irFetcherWithNotary) InnerRingKeys() (keys.PublicKeys, error) {
+	return fN.cli.NeoFSAlphabetList()
+}
+
+type irFetcherWithoutNotary struct {
+	nm *nmWrapper.Wrapper
+}
+
+// InnerRingKeys fetches list of innerring keys from netmap contract
+// in side chain.
+func (f irFetcherWithoutNotary) InnerRingKeys() (keys.PublicKeys, error) {
```

Comment:
Move all fetcher definitions to `indexer.go`.

Reply:
https://github.com/nspcc-dev/neofs-node/pull/705/commits/40a6ea7caf67e04d48d9df58103d259c90bd369d